versioned_docs/version-2.x/advanced/images-adapters/image-component.mdx

```diff
@@ -0,0 +1,94 @@
+---
+sidebar_position: 5
+title: Image component
+description: Use Front-Commerce's image component to display your images
+---
+
+import SinceVersion from "@site/src/components/SinceVersion";
+
+<p>{frontMatter.description}</p>
+
+The Front-Commerce `Image` component allow you to display and cache optimized
```

Comment:
**issue:** we already had some documentation about the `<Image />` component in https://developers.front-commerce.com/docs/2.x/advanced/production-ready/media-middleware#image-component

This section of the documentation was for Image adapters only,

**suggestion:** I don't have a strong opinion about the target. Maybe having this dedicated page is a good move.\
In that case, we should rework the existing page's section to only provide a simple and realistic example and link to this detailed page.

There might also be existing content from the documentation that will need to be added to this page.

Reply:
IMO : We should create a page for specific FC Components (like Image), where we can put some documentation (excluding simple atoms/molecules/organisms).